docs/Fault-Tolerant-Tachyon-Cluster.md

```diff
@@ -82,5 +98,5 @@ For a worker, it is only necessary to set the `TACHYON_MASTER_ADDRESS` option as
 
     export TACHYON_MASTER_ADDRESS=[address of one of the master nodes in the system]
 
-Any of the configured and running masters can be used, as they will inform the worker of the current
-leader.
+Any master address can be used, when configured to use Zookeeper workers will look up the
```

Comment:
This section is describing the worker configuration so I believe this to be appropriate wording here